### PR TITLE
Resolved sf syst update

### PR DESCRIPTION
**Pull request by pastika**

### CODE REVIEW

**Comment by pastika** [comment]
@zhenbinwu This PR is ready for review

File: python/processors/Stop0l_postproc.py [hunk 1]
```diff
@@ -62,76 +65,96 @@
...
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2018_v1.0.2",
                    },
         "2018D" : { "JEC": "Autumn18_RunD_V8_DATA",
                     "redoJEC": True,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2018_v1.0.2",
                    },
             }
 }
 
-DeepResovledDiscCut = 0.6
```
Comment (zhenbinwu) [inline python/processors/Stop0l_postproc.py]: Why are we changing the cut?

File: README.md [hunk 2]
```diff
@@ -17,14 +17,16 @@ cd $CMSSW_BASE/src
...
 scram b
 cd $CMSSW_BASE/src/TopTagger/TopTagger/test
 ./configure
 make
 cmsenv
 cd $CMSSW_BASE/src/PhysicsTools/NanoSUSYTools/python/processors
-getTaggerCfg.sh -t DeepResolved_DeepCSV_GR_nanoAOD_V1.0.0
+getTaggerCfg.sh -n -t DeepResolved_DeepCSV_GR_nanoAOD_2016_v1.0.2
+getTaggerCfg.sh -n -t DeepResolved_DeepCSV_GR_nanoAOD_2017_v1.0.2
+getTaggerCfg.sh -n -t DeepResolved_DeepCSV_GR_nanoAOD_2018_v1.0.2
```
Comment (zhenbinwu) [inline README.md]: So these tags for just the systematic right? But we are using the same NN for all 3 years?

File: python/processors/Stop0l_postproc.py (hunk 1 above)
Comment (pastika) [reply python/processors/Stop0l_postproc.py]: The final analysis cut is at 0.92, so this just cuts out a few unused tops to offset the space added by the systematics .

File: README.md (hunk 2 above)
Comment (pastika) [reply README.md]: Correct.

**Comment by zhenbinwu** [comment]
ok, merging

**Pull request merged**